src/Abp/Authorization/AuthorizeAttributeHelper.cs

```diff
@@ -16,11 +19,13 @@ public AuthorizeAttributeHelper()
         {
             AbpSession = NullAbpSession.Instance;
             PermissionChecker = NullPermissionChecker.Instance;
+            MultiTenancyConfig = new MultiTenancyConfig();
```

Comment:
OK, now understood. Since it has no null-object, constructor injection would be better.